test_programs/compile_success_empty/trait_call_in_global/src/main.nr

```diff
@@ -1,3 +1,5 @@
 global s: BoundedVec<Field, 2> = From::from([0]);
 
-fn main() {}
+fn main() {
+    let _ = s;
```

Comment:
You could also rename `s` to `_s` if you'd prefer that

Reply:
You mean, removing the `let`?

Reply:
Yes, since you made it to get rid of the unused global warning I assume? Your choice, I was just pointing it out as an option.

Reply:
The thing is, if I remove this `let` statement then the program compiles fine in master. If it's there it gives an error. I think it's because the global variable is used in this case and some typing happens, not sure.

Originally the program had `println(s)` but that fails inside `compile_success_empty` and I didn't know where else to put it like that.